ignite/contrib/engines/common.py

```diff
@@ -1,25 +1,27 @@
-from functools import partial
-import warnings
 import numbers
-
-from collections.abc import Sequence, Mapping
+import warnings
+from collections.abc import Mapping, Sequence
+from functools import partial
 
 import torch
 import torch.distributed as dist
 
-from ignite.engine import Engine, Events
-from ignite.metrics import RunningAverage
-from ignite.handlers import TerminateOnNan, ModelCheckpoint, EarlyStopping
-from ignite.contrib.metrics import GpuInfo
-from ignite.contrib.handlers import ProgressBar
-from ignite.contrib.handlers import VisdomLogger
-from ignite.contrib.handlers import TensorboardLogger, global_step_from_engine
-import ignite.contrib.handlers.tensorboard_logger as tb_logger_module
-import ignite.contrib.handlers.visdom_logger as visdom_logger_module
-from ignite.contrib.handlers import MLflowLogger
 import ignite.contrib.handlers.mlflow_logger as mlflow_logger_module
-from ignite.contrib.handlers import PolyaxonLogger
 import ignite.contrib.handlers.polyaxon_logger as polyaxon_logger_module
+import ignite.contrib.handlers.tensorboard_logger as tb_logger_module
+import ignite.contrib.handlers.visdom_logger as visdom_logger_module
+from ignite.contrib.handlers import (
```

Comment:
@sisp maybe we can rename above modules and reuse in the code:
```
import ignite.contrib.handlers.mlflow_logger as mlf_module
import ignite.contrib.handlers.polyaxon_logger as plx_module
import ignite.contrib.handlers.tensorboard_logger as tb_module
import ignite.contrib.handlers.visdom_logger as vd_module
```
such that we do not need to import `MLflowLogger`, `PolyaxonLogger`, etc and keep only `ProgressBar`, `global_step_from_engine`.

Reply:
Yes, we could do it like you suggested.

Just a reference to my general remark about how the modules are used there to connect the discussions: https://github.com/pytorch/ignite/pull/901#issuecomment-612414100